routes/user.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+const router = express.Router();
+const { check, validationResult } = require("express-validator");
+require("dotenv").config({ path: "../.env" });
+
+const sgMail = require("@sendgrid/mail");
+const User = require("../models/User");
+
+sgMail.setApiKey(process.env.EMAIL_KEY);
+
+//POST create a user invitation
+router.post("/:id/invitation", async (req, res) => {
```

Comment:
Once we get the authentication work finished, we'll want this to be a protected route

Reply:
would using an auth middleware route be a good option?

Reply:
yes, definitely!